src/diff/diffCommand.ts

```diff
@@ -161,12 +181,12 @@ Exit statuses:
     const apiDifferencer = new ApiDifferencer(baseApis, newApis);
     try {
       apiChanges = await apiDifferencer.findAndCategorizeChanges(flags.ruleset);
-      await this._saveOrLog(apiChanges, flags["out-file"]);
+      await this._saveOrLog(apiChanges, flags);
     } catch (err) {
       this.error(err.message, { exit: 2 });
     }
     if (apiChanges.hasBreakingChanges()) {
-      this.error("Breaking changes found.", { exit: 1 });
+      this.exit(1);
```

Comment:
Why this change?

Reply:
The new formatted output tells you whether or not breaking changes are found, so the extra output is not necessary.